.gitpod.yml

```diff
@@ -3,14 +3,13 @@ tasks:
     env:
       WOODPECKER_OPEN: true
       WOODPECKER_ADMIN: woodpecker
-      WOODPECKER_HOST: http://host.docker.internal:8000
       WOODPECKER_EXPERT_WEBHOOK_HOST: http://host.docker.internal:8000
       WOODPECKER_AGENT_SECRET: '1234'
       WOODPECKER_GITEA: true
       WOODPECKER_DEV_WWW_PROXY: http://localhost:8010
       WOODPECKER_BACKEND_DOCKER_NETWORK: ci_default
     init: |
-      GO_VERSION=1.22
+      GO_VERSION=1.24.2
```

Comment:
```suggestion
      # renovate: datasource=golang-version depName=golang
      GO_VERSION=1.24.2
```

Reply:
I prepared the renovate part https://github.com/woodpecker-ci/renovate-config/pull/31